### ISSUE TITLE
More of a question for assistance with implementing a search for navigation.

### ISSUE THREAD

**Issue by s3venvials** [LeaVerou/awesomplete#17149]
Was needing assistance with getting this to work with navigating to URL's where the label is say Google but upon enter key or button click it would navigate to the corresponding URL. I have an almost working example here. https://github.com/s3venvials/my-awesomplete. Any assistance is appreciated.

**Comment by LeaVerou** [comment]
You mean navigating somewhere when an option gets selected? Have you looked at the events we support? http://leaverou.github.io/awesomplete/#events

**Comment by s3venvials** [comment]
Thanks LeaVerou for your response and yes but I did not want the event to fire upon selection from the drop down. I wanted the event to fire upon enter or button click. I figured it out though, I was making this out to be more complicated than it needed to be. Great looking autocomplete! I was using the UI-Autocomplete from jQuery but it did not seem to work well with my project. This looks and works awesome! :)
